src/components/NavBar/MenuDropdown.tsx

```diff
@@ -48,7 +47,7 @@ const PrimaryMenuRow = ({
           <Row onClick={close}>{children}</Row>
         </NavLink>
       ) : (
-        <Row as="a" href={href} target="_blank" rel="noopener noreferrer" className={styles.MenuRow}>
+        <Row cursor="pointer" as="a" href={href} target="_blank" rel="noopener noreferrer" className={styles.MenuRow}>
```

Comment:
fixes this tiny bug since I'm touching this file:

<img width="482" alt="image" src="https://user-images.githubusercontent.com/39385577/233679022-99c7aa9c-9493-488f-8507-81c04a86f5be.png">

Reply:
Should the `cursor="pointer" also be removed from `SecondaryLinkedText`?